mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/events/LifecycleEventPublisherImpl.java

```diff
@@ -38,10 +40,14 @@ public void publishAuditEvent(final LifecycleEventsProto.AuditEvent auditEvent)
 
     @Override
     public void publishStatusEvent(final LifecycleEventsProto.StatusEvent statusEvent) {
-        statusEventSubscriber.process(statusEvent);
-        if (statusEvent instanceof LifecycleEventsProto.JobStatusEvent) {
-            LifecycleEventsProto.JobStatusEvent jobStatusEvent = (LifecycleEventsProto.JobStatusEvent) statusEvent;
-            workerEventSubscriber.process(jobStatusEvent);
+        try {
+            statusEventSubscriber.process(statusEvent);
+            if (statusEvent instanceof LifecycleEventsProto.JobStatusEvent) {
+                LifecycleEventsProto.JobStatusEvent jobStatusEvent = (LifecycleEventsProto.JobStatusEvent) statusEvent;
+                workerEventSubscriber.process(jobStatusEvent);
+            }
+        } catch (Exception e) {
+            log.error("Failed to publish the event; Ignoring the failure as this is just a listener interface", e);
```

Comment:
is the statusEvent printable? can we add it to the log line?

Reply:
done.